cmd/security-secretstore-setup/res/configuration.toml

```diff
@@ -39,6 +39,8 @@ TokenProvider = "/security-file-token-provider"
 TokenProviderArgs = [ "-confdir", "res-file-token-provider" ]
 TokenProviderType = "oneshot"
 TokenProviderAdminTokenPath = "/run/edgex/secrets/tokenprovider/secrets-token.json"
+PasswordProvider = "openssl"
+PasswordProviderArgs = [ "rand", "-base64", "36" ]
```

Comment:
In fact, I put this in for testing and forgot to remove it.

`PasswordGeneratorContextOption` didn't seem very useful, since Redis only supports and single password and mongo is going to be deprecated.  neither openssl nor apg support a context parameter, so it seemed to be an extra feature with no immediate way to put it into use.